### PR TITLE
Bugfixes: Fax machines, CentCom turrets, Robo-limb emag. New feature for monkey cubes

### PR DESCRIPTION
**Pull request by inselc**
This PR contains bugfixes for:
- Fax machine cooldown
- CCIAA turret whitelist (possibly also fixes related access issues)
- Animation playing while "sneakily" emagging robotic limbs

While at it, added feature to broadcast fax to all on-station fax machines.

Added ability to expand monkey cubes using water tanks, since that one was a single-line-mod. Is it alright to include that one in this PR (since it's not really a bugfix)?

### CODE REVIEW

File: code/modules/paperwork/faxmachine.dm [hunk 1]
```diff
@@ -169,9 +181,19 @@ var/list/sent_faxes = list()	//cache for faxes that have been sent by the admins
 
 	updateUsrDialog()
 
-/obj/machinery/photocopier/faxmachine/proc/sendfax(var/destination)
+// Only run this in a spawn()!
+/obj/machinery/photocopier/faxmachine/proc/process_cooldown()
```
Comment (skull132) [inline code/modules/paperwork/faxmachine.dm]: This should be worked into the fax machine's process() proc instead. delay(), spawn(), or sleep() shouldn't be used in a process-like proc, as they remove control over scheduling from the actual ingame controllers.

You'll need to save the world.time of last process, deduct that from current process to get delta time, and then deduct delta time from sendcooldown. Also make sure that `. = ..()` is in there.

File: code/modules/paperwork/faxmachine.dm [hunk 2]
```diff
@@ -1,6 +1,7 @@
 var/list/obj/machinery/photocopier/faxmachine/allfaxes = list()
 var/list/admin_departments = list("Central Command", "Sol Government")
 var/list/alldepartments = list()
+var/broadcast_departments = "Stationwide broadcast (WARNING)"
```
Comment (skull132) [inline code/modules/paperwork/faxmachine.dm]: Does this have to be a global var? It could either be a class constant or a proc static constant.

Comment (skull132) [inline code/modules/paperwork/faxmachine.dm]: Edit, class static might be the most ideal.

File: code/modules/paperwork/faxmachine.dm (hunk 1 above)
Comment (inselc) [inline code/modules/paperwork/faxmachine.dm]: Amended in fb57de624b267af4bf3366a7d38409cc604060c2, but used a slightly different approach to determine the remaining cooldown time.

File: code/modules/paperwork/faxmachine.dm (hunk 2 above)
Comment (inselc) [inline code/modules/paperwork/faxmachine.dm]: Thanks for the heads-up. Makes sense to make these vars static class members. Amended in fb57de624b267af4bf3366a7d38409cc604060c2.

**Pull request merged**